clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -895,4 +895,8 @@ def err_drv_triple_version_invalid : Error<
 
 def warn_missing_include_dirs : Warning<
   "no such include directory: '%0'">, InGroup<MissingIncludeDirs>, DefaultIgnore;
+
+def warn_compress_opt_ignored : Warning<
+  "CLI option %0 ignored as it can not be used without -offload-compress">,
```

Comment:
```suggestion
  "CLI option '%0' ignored as it can not be used without '-offload-compress'">,
```

Reply:
Fixed in 64878676eaff6c6c62a086b1aa41b0e2752a4b91